modules/kmm-hub-installing-kmm-hub-kubectl.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-hub-installing-kmm-hub-kubectl_{context}"]
+= Installing KMM-Hub using `kubectl`
+
+Use this procedure to install KMM-Hub using `kubectl`.
+
+.Procedure
```

Comment:
This is the procedure for U/S .

CertManager is required for Upstream version of KMM-HUB

Reply:
This section has been removed.